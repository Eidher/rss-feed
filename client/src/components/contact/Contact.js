import React, { Component } from "react";

class Contact extends Component {
  render() {
    return (
      <section id="contact">
        <div className="header">Thank you!</div>
        <div className="main">
          <div className="name">Eidher Escalona</div>
          <div className="footer">eidher.escalona@gmail.com</div>
        </div>
      </section>
    );
  }
}

export default Contact;
