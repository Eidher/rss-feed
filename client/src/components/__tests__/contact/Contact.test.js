import React from "react";
import { shallow } from "enzyme";
import Contact from "../../contact/Contact";

describe("<Contact />", () => {
  const wrapper = shallow(<Contact />);

  it("renders correctly.", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
