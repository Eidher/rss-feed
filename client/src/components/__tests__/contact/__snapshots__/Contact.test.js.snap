// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact /> renders correctly. 1`] = `
<section
  id="contact"
>
  <div
    className="header"
  >
    Thank you!
  </div>
  <div
    className="main"
  >
    <div
      className="name"
    >
      Eidher Escalona
    </div>
    <div
      className="footer"
    >
      eidher.escalona@gmail.com
    </div>
  </div>
</section>
`;
